controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -69,7 +86,7 @@ func (r *StorageClassClaimReconciler) Reconcile(ctx context.Context, request rec
 	var result reconcile.Result
 	var reconcileError error
 	if storagecluster.IsOCSConsumerMode(&storageClusterList.Items[0]) {
-		result, reconcileError = r.reconcileConsumerPhases(ctx, &storageClusterList.Items[0])
+		result, reconcileError = r.reconcileConsumerPhases(ctx, &storageClusterList.Items[0], claim)
```

Comment:
What if the storage cluster list is empty? we cannot directly address index 0 in that case.